### PR TITLE
Saiyyed

### PR DESCRIPTION
**Pull request by ziyaulhuque**

### CODE REVIEW

**Comment by ziyaulhuque** [comment]
Changes are done

File: src/test/java/pageObjects/ForgetLoginInfoPage.java [hunk 1]
```diff
@@ -6,33 +7,44 @@
...
+import org.testng.Assert;
 
-public class ForgetLoginInfoPage {
 
+public class ForgetLoginInfoPage {
     @FindBy(linkText = "Forgot login info?")
-    public WebElement fr_login;
-    @FindBy(xpath = "//*[@id=\"rightPanel\"]/h1")
-    public WebElement title;
-    @FindBy(xpath = "//*[@id=\"rightPanel\"]/p")
```
Comment (JPrathmesh) [inline src/test/java/pageObjects/ForgetLoginInfoPage.java:18]: Remove the \ (slash) from xpath

File: src/test/java/pageObjects/ForgetLoginInfoPage.java [hunk 2]
```diff
@@ -42,69 +54,76 @@ public ForgetLoginInfoPage(WebDriver driver) {
...
-        System.out.println(info_title.getText());
-        Assert.assertEquals(info_title.getText(), title2);
+        System.out.println(lblMsgTitle.getText());
+        Assert.assertEquals(lblMsgTitle.getText(), title2);
+        Reporter.log("get Title" + title2);
+
+
     }
 
     public void firstName(String first_name) {
```
Comment (JPrathmesh) [inline src/test/java/pageObjects/ForgetLoginInfoPage.java]: use enter word on creating  method

**Review by JPrathmesh — CHANGES_REQUESTED** [review]
Changes Requested

**Comment by ziyaulhuque** [comment]
Final changes are done from saiyyed in forgetinfopage.

**Review by JPrathmesh — APPROVED** [review]

**Review by JPrathmesh — APPROVED** [review]

**Pull request merged**